src/index.ts

```diff
@@ -41,17 +42,30 @@ const port = 8080
 // extract json payload from body automatically
 app.use(bodyParser.json({ limit: '1mb' }))
 
+// function to exlude paths from middleware
+const unless = function (path, middleware) {
+    return function (req, res, next) {
+        if (path === req.path) {
+            return next()
+        } else {
+            return middleware(req, res, next)
+        }
+    }
+}
+
 // throw exception if not authorized
-app.use(authMiddleware)
+app.use(unless('/health', authMiddleware))
 
 // throw error if no payload submitted
-app.use(function (req: Request, res: Response, next: Function) {
-    if (Object.keys(req.body).length === 0) {
-        const err = new PayloadError('Request format invalid')
-        return next(err)
-    }
-    return next()
-})
+app.use(
+    unless('/health', function (req: Request, res: Response, next: Function) {
+        if (Object.keys(req.body).length === 0) {
+            const err = new PayloadError('Request format invalid')
+            return next(err)
+        }
+        return next()
+    })
+)
```

Comment:
```suggestion
app.use(
    unless('/health', function (req: Request, res: Response, next: Function) {
        if (Object.keys(req.body).length === 0) {
            const err = new PayloadError('Request format invalid')
            return next(err)
        }
        return next()
    })
)
```

Not perfect, but it fixes the `tsc` errors.